File: Python/minutes_into_years_and_days.py
def main(x):
    years=x/(365*24*60)
    days=(x/24/60)%365
    print("%d minutes is approximately" %x, "%d years and" %years ,"%d days" %days)

if __name__ == '__main__':
    print("Enter the number of minutes: ")
    minutes=int(input())
    main(minutes)

File: Java/minutes_into_years_and_days.java
import java.util.Scanner;
public class minutes_into_years_and_days {

    public static void main(String[] Strings) {


        double minutesInAYear = 365 * 24 * 60;

        Scanner scan = new Scanner(System.in);

        System.out.print("Input the number of minutes: ");

        double min = scan.nextDouble();
        scan.close();

        long years = (long) (min / minutesInAYear);
        int days = (int) (min / 60 / 24) % 365;

        System.out.println((int) min + " minutes is approximately " + years + " years and " + days + " days");
    }
}